### ISSUE TITLE
Atari roms broken?

### ISSUE THREAD

**Issue by maraghuram** [ppwwyyxx/tensorpack#523]
I downloaded rom file for ms_pacman from the url mentioned here: https://github.com/ppwwyyxx/tensorpack/blob/130f60ac4491d55a13632eabbad839c2e23138ba/examples/DeepQNetwork/atari.py#L25
This prints errors like "Illegal Instruction! 32" at line 91 same file and does not render anything. But strangely gym.make('MsPacman-v0') works. So I looked at the atari_py repo and saw that the roms have not been updated in 2 years.

TL;DR: the roms on the github repo seem to be broken for a few games. Roms present locally at '/site-packages/atari_py/atari_roms' are working at the moment. You might want to update the debug message to download from the url to something else. 

Thanks

**Comment by ppwwyyxx** [comment]
Where do the roms in "site-packages/atari_py/atari_roms" come from? I think they came from exactly the same place.

**Comment by maraghuram** [comment]
Sorry my bad. They are identical. 
I was doing
`wget https://github.com/openai/atari-py/blob/master/atari_py/atari_roms/ms_pacman.bin` instead of `wget https://github.com/openai/atari-py/raw/master/atari_py/atari_roms/ms_pacman.bin`

Thanks for the quick response.
